src/mesh/abstract_tree.jl

```diff
@@ -576,8 +593,8 @@ end
 # Raw copy operation for ranges of cells.
 #
 # This method is used by the higher-level copy operations for AbstractContainer
-function raw_copy!(target::AbstractTree, source::AbstractTree, first::Int, last::Int, destination::Int) end
+# function raw_copy!(target::AbstractTree, source::AbstractTree, first::Int, last::Int, destination::Int) end
```

Comment:
why commented (also `reset_data_structures!` below)? I think I added those here such that they can be extended with methods by other modules, back when the tree was still its own module. If  this is not required anymore, we can just change it as you proposed (maybe this is also the reason for all the other commented functions?)

Reply:
See `refine_unbalanced!`